indico/htdocs/js/indico/modules/eventmanagement.js

```diff
@@ -0,0 +1,41 @@
+(function(global) {
+    'use strict';
+
+    global.setupEventPersonsList = function setupEventPersonsList() {
```

Comment:
`setupContributionPersonsList`? it's specific to contribution person links because of the speaker/author stuff (event person links are just links without extra data, same for subcontribution person links)